CHANGELOG.md

```diff
@@ -37,11 +37,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Added EFM32TG11B family targets #1420
+
 - Added LPC55Sxx target #1513
+
 - Added custom sequence support to STM32L0, L1, L4, G0, G4, F0, F3, WB, WL,
   enabling debug clocks during sleep modes #1521
+
 - Add default sequence 'debug_core_stop', which disables debugging when disconneting from ARM cores by default. (#1525)
 
+- probe-rs-debugger: Add support (Part 1) for 'gdb-like' commands to be typed into VSCode Debug Console REPL. ()
```

Comment:
Part 1 should not really be mentioned here, but rather what the limitations are and what is planned if anything :)

Reply:
Fair enough ... I added some clarification :)